### PR TITLE
Added Contact Us Page

### PR DESCRIPTION
**Pull request by Hemashree21**
I added the UI for the Contact Us page and it's responsive as well.

image

@Ajay-Dhangar please check and let me know if any changes are required

### CODE REVIEW

File: src/pages/contact/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Layout from "@theme/Layout";
+import styles from './styles.module.css';
+
+/**
+ * Contact Us Page
+ */
+
+const Index = () => {
```
Comment (Ajay-Dhangar) [inline src/pages/contact/index.tsx:9]: you can use or replace `Contact` to `Index`

File: src/pages/contact/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <textarea placeholder='Your message...' className={styles.mytextarea} name="message" />
+        <button className={styles.inputsubmit} type="submit">Send</button>
+      </form>
+      <div className={styles.image}><img src="https://cdn-icons-png.flaticon.com/512/5075/5075578.png" alt="" style={{height: '25rem'}}/></div>
+      </div>
+    </Layout>
+  )
+}
+
+export default Index;
```
Comment (Ajay-Dhangar) [inline src/pages/contact/index.tsx:29]: you can use or replace `Contact` to `Index`

File: src/pages/contact/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
+.myform {
```
Comment (Ajay-Dhangar) [inline src/pages/contact/styles.module.css:1]: use grid for layout for good responsive design

Comment (Ajay-Dhangar) [reply src/pages/contact/styles.module.css:1]: Because something is not working 

image

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Now Change code based on suggestions

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by Hemashree21** [comment]
Thankyou

**Pull request merged**